src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  src: url('https://fonts.googleapis.com/css?family=Roboto:500&display=swap');
```

Comment:
The `src` property is not valid in CSS and should be removed. Font imports should be handled in the HTML or using `@import` in CSS.